### PR TITLE
use non-root location

### PR DESCRIPTION
**Pull request by satra**
change to use a non-root location to help with singularity

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -25,7 +25,7 @@ dependencies:
 test:
   override:
     # Test mriqcp
-    - docker run -i -v /etc/localtime:/etc/localtime:ro -v ${CIRCLE_TEST_REPORTS}:/scratch -w /root/src/mriqc --entrypoint="/usr/bin/run_tests" mriqc:py35 :
+    - docker run -i -v /etc/localtime:/etc/localtime:ro -v ${CIRCLE_TEST_REPORTS}:/scratch -w /opt/src/mriqc --entrypoint="/usr/bin/run_tests" mriqc:py35 :
```
Comment (satra) [inline circle.yml]: @oesteban - do you want the workdir to be /src/mriqc or sth like /workdir ?

Comment (oesteban) [inline circle.yml]: I don't have a preference, it is ok right now

**Comment by chrisfilo** [comment]
Interesting. Does Singularity mount /root on your system?

**Comment by satra** [comment]
singularity shows a root folder but as a user we don't have access.

**Comment by chrisfilo** [comment]
So this is more of permissions issue then. MRIQC operates under root user so even if you put the code in /opt it might not solve the issue.

Both [docker2singularity ](https://github.com/singularityware/docker2singularity) and [docker2singularity.py](https://github.com/oschulz/singularity-utils/blob/master/bin/docker2singularity.py) change permissions when importing containers from docker hub to avoid such problems.

**Comment by satra** [comment]
the docker user doesn't matter in singularity context, but `/root` is special.

**Comment by chrisfilo** [comment]
Doesn't Dockerfile user influence the ownership of files in the container image deposited on Docker Hub which would be a problem when those files are downloaded by singularity?

**Comment by satra** [comment]
not really, because we are using singularity as an executable effectively, so as long as things are readable we are ok. now if you try to write to some location that's not accessible to the user, they one can run into issues, but containers should be very clear as to where they are writing.

**Pull request merged**